app/views/layouts/base.html.erb

```diff
@@ -15,7 +15,15 @@
   <meta content="noindex,nofollow" name="robots" />
   <% end %>
 
-  <title><%= yield(:title).presence || (raise 'Missing title') %> | <%= APP_NAME %></title>
+  <%
+    title = begin
+      yield(:title).presence || (raise 'Missing title (error hidden from user)')
+    rescue => err
+      NewRelic::Agent.notice_error(err)
+      ''
+    end
```

Comment:
I think I'd rather have the option to still raise (e.g. dev and test). It probably doesn't make sense to do something as complicated as the following in the template though (maybe a ViewComponent?). I think avoiding the raise/rescue and calling NewRelic directly would be preferable if possible too.

```suggestion
    title = yield(:title).presence
    if title.nil? && IdentityConfig.store.raise_on_missing_title
      raise 'Missing title (error hidden from user)')
    else
      NewRelic::Agent.notice_error(err)
    end
```

Reply:
I wanted the `raise` to make sure it gets the backtrace. I'll look into the config change

```
irb(main):001> x = RuntimeError.new('aaa')
=> #<RuntimeError: aaa>
irb(main):002* begin
irb(main):003*   raise 'bbb'
irb(main):004* rescue => e
irb(main):005> end
=> nil
irb(main):006> x.backtrace
=> nil
irb(main):007> e.backtrace
=> 
["(irb):3:in `<top (required)>'",
```

Reply:
PTAL at ba431735b037acccb5744eb48c97a542381f06c4